Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLink.cs

```diff
@@ -765,15 +767,21 @@ public void SendAudioOutputData()
                 }
                 else
                 {
-                    audioSource.GetOutputData(_audioFramesR, 1);
+                    if (VRC.SDKBase.Utilities.IsValid(optionalRightAudioSource))
+                    {
+                        optionalRightAudioSource.GetOutputData(_audioFramesR, 0);
+                    } else audioSource.GetOutputData(_audioFramesR, 1);
                 }
                 _rightChannelTestCounter--;
             }
             else
             {
-                _rightChannelTestCounter = _rightChannelTestDelay;      // reset test countdown
-                _audioFramesR[0] = 0f;                                  // reset tested array element to zero just in case
-                audioSource.GetOutputData(_audioFramesR, 1);            // right channel test
+                _rightChannelTestCounter = _rightChannelTestDelay;                  // reset test countdown
+                _audioFramesR[0] = 0f;                                              // reset tested array element to zero just in case
+                if (VRC.SDKBase.Utilities.IsValid(optionalRightAudioSource))        // check if dual mono is present
```

Comment:
issue (blocking): any calls to VRC namespace stuff should be wrapped with `#if UDONSHARP` or `#if VRC_SDK_VRCSDK3` depending on if that codepath should be taken in Avatar Projects, in this case I believe it should be `UDONSHARP`